system/jlib/jtrace.cpp

```diff
@@ -1857,6 +1857,14 @@ ISpan * CTraceManager::createServerSpan(const char * name, const IProperties * h
     return new CServerSpan(name, httpHeaders, flags, spanStartTimeStamp);
 }
 
+ISpan * createCompletedInternalSpan(const char * name, stat_type elapsedNs)
```

Comment:
I like the concept, but I see nothing inherently "completed" about this span, I think it relies on the immediate release by the caller. I recommend we rename to avoid misunderstanding. That said, I'm not sure what would be a more appropriate name... "createDelayedInternalSpan" or "createInternalSpanAndAdjustStartTime"